backend/eventloop/db_reconciler.go

```diff
@@ -1102,11 +1107,11 @@ func getListOfCRIdsFromTable(ctx context.Context, dbQueries db.DatabaseQueries,
 			// Save list of IDs in map according to resource type.
 			for _, deplToAppMapping := range tempList {
 				if deplToAppMapping.DBRelationType == db.APICRToDatabaseMapping_DBRelationType_RepositoryCredential {
-					crIdMap[dbType_RespositoryCredential] = append(crIdMap[dbType_RespositoryCredential], deplToAppMapping.DBRelationKey)
+					crIdMap[dbType_RespositoryCredential][deplToAppMapping.DBRelationKey] = false
```

Comment:
This is a nit, but when you reference the map, you use `_, ok := ....` so you don't check the value.  So, to maintain consistency, then should these be assigned `true` ?  That way you can check the existence by way of the key/id, or by the value?